tests/e2e/kubetest2-kops/builder/build.go

```diff
@@ -21,20 +21,28 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 
+	rbuild "k8s.io/release/pkg/build"
+
 	"k8s.io/klog/v2"
 	"sigs.k8s.io/kubetest2/pkg/exec"
 )
 
 type BuildOptions struct {
-	KopsRoot      string `flag:"-"`
-	StageLocation string `flag:"-"`
+	KopsRoot        string `flag:"-"`
+	KubeRoot        string `flag:"-"`
```

Comment:
I remembered something, specifying this doesn't matter as the cwd must be the k/k folder, it will break if you call the make commands from a different folder.(badly coded scripts)